book/theme/css/variables.css

```diff
@@ -144,6 +159,13 @@
     --search-mark-bg: #a2cff5;
 
     --color-scheme: light;
+
+    /* Same as `--icons` */
+    --copy-button-filter: invert(45.49%);
+    /* Same as `--sidebar-active` */
+    --copy-button-filter-hover: invert(14%) sepia(93%) saturate(4250%) hue-rotate(243deg) brightness(99%) contrast(130%);
```

Comment:
This seems really complicated when we could just assign a hex value?

Reply:
This is pulled from the official source code - i wouldn't want to mess with it haha

Reply:
The TOC will be created automatically ... it's an addition to the theming